.github/workflows/integrationTest.yml

```diff
@@ -297,12 +299,69 @@ jobs:
           -var="arc=${{ matrix.arrays.arc }}"
           -var="binary_name=${{ matrix.arrays.binaryName }}"
           -var="local_stack_host_name=${{ needs.StartLocalStack.outputs.local_stack_host_name }}"
+
       - name: Terraform destroy
         if: ${{ always() && steps.ec2-linux-integration-test.outputs.cache-hit != 'true' }}
         run: >
           terraform destroy --auto-approve
           -var="ami=${{ matrix.arrays.ami }}"
 
+  EC2WinIntegrationTest:
+    needs: [BuildMSI]
+    name: 'EC2WinIntegrationTest'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: integration/terraform/ec2/win
+    strategy:
+      fail-fast: false
+      matrix:
+        arrays: [
+          { os: "win-2022", ami: "cloudwatch-agent-integration-test-win-2022*"}
```

Comment:
Isn't there an outstanding bug for windows 2022 that needs to be fixed? Or did I miss the fix PR for that? Was under the assumption that we would still be targeting 2019.

Reply:
This is for the test of @ZhenyuTan-amz fix.